app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
```

Comment:
Maybe `An upgrade (or update) is the process of changing the version of the software currently deployed to a more recent one. We suggest to follow the steps below to perform an effective upgrade; notice, however, that this approach will require downtime. In case you need high availability, review the Blue/Green Deployment section below.`?